expr/js_eval.go

```diff
@@ -215,15 +214,16 @@ func EvaluateResultType[ReturnType any](ctx context.Context, source string, resu
 
 	case result.IsUndefined():
 		// do nothing, undefined gets skipped
+		return resultValue, nil
+
+	case isArray(result):
+		fmt.Fprintf(os.Stdout, "\n  Source %s evaluates to an array. Handling separately\n", source)
```

Comment:
Do we need to log anything here? I'm not sure we should. I think we'll only hit this when we try to evaluate an alias JS expression as a string but it returns an array, and the user doesn't need to know about that.

Reply:
we're catering for our known path but a user could write all sorts of bad configuration , e.g.
`name: $.aliases`

in this case we don't error but it's worth making some level of noise imo